mesop/components/button_toggle/button_toggle.ng.html

```diff
@@ -0,0 +1,18 @@
+<mat-button-toggle-group
+  [style]="getStyle()"
+  [hideMultipleSelectionIndicator]="config().getHideMultipleSelectionIndicator()"
+  [hideSingleSelectionIndicator]="config().getHideSingleSelectionIndicator()"
```

Comment:
i don't understand why Angular Material's API separates this out into two properties vs. just having one like `hideSelectionIndicator` because there's already a separate `multiple` property.

Am I missing something about why these should be separate properties? Should we simplify this in our API?

Reply:
Great point. Yeah I don't really know what the use case for it would be. Updated to combine them.